### PR TITLE
About Page (Mission Section and Timeline Styled)

### PR DESCRIPTION
**Pull request by T-Joseph-Kim**
image
image
image

### CODE REVIEW

File: app.config.timestamp_1729437130451.js
Comment (ufsasewebmaster) [inline app.config.timestamp_1729437130451.js:1]: Remove this file

File: client/components/styles/timeline.css
Comment (ufsasewebmaster) [inline client/components/styles/timeline.css:1]: We are using tailwind, not regular CSS. Make sure you're considering the readability and integration of the code.

File: tailwind.config.cjs
Comment (ufsasewebmaster) [inline tailwind.config.cjs:1]: You can define Poppins in the tailwind config instead of here.

**Review by ufsasewebmaster — CHANGES_REQUESTED** [review]
See comments

File: client/index.css
Comment (ufsasewebmaster) [inline client/index.css:1]: You can remove the Poppins fontfamily since it's already in the tailwind config

Comment (T-Joseph-Kim) [reply client/index.css:1]: In order to do use the font correctly, I will have to leave it in index.css and remove it in tailwind config.

Comment (ufsasewebmaster) [reply client/index.css:1]: Wait why? We're using tailwind so shouldn't you be able to define it only in the tailwind config?

Comment (T-Joseph-Kim) [reply client/index.css:1]: I honestly am not sure. There is definitely a way to do it through tailwind config. Index.css allows me to use the font properly with weights. When I import it in tailwind config, the font is always bold which means I'm not actually using it from tailwind config. I will try and figure this out if its better to do it through tailwind config.

**Comment by T-Joseph-Kim** [comment]
Added contact us form and added many styling changes. Still need a new timeline that incorporates tailwind css.

**Comment by ufsasewebmaster** [comment]
I fixed some conflicts by merging with sprint-2, but just in case check that everything you made is still present. A couple small things then it will be good:
Make the timeline scrolling a little more intuitive
Host the constitution pdf locally instead of from WP
Fix formatting checks (I forgot to do them)

**Pull request merged**